src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -972,6 +974,28 @@ EnsureReplicationSettings(Oid relationId, char replicationModel)
 }
 
 
+/*
+ * EnsureRelationHasNoTriggers errors out if the given table has triggers on
+ * it. See also GetExplicitTriggerOids function's comment for the triggers this
+ * function errors out.
+ */
+static void
+EnsureRelationHasNoTriggers(Oid relationId)
+{
+	List *explicitTriggerIds = GetExplicitTriggerOids(relationId);
+
+	if (list_length(explicitTriggerIds) > 0)
+	{
+		char *relationName = get_rel_name(relationId);
+
+		ereport(ERROR, (errmsg("cannot distribute relation \"%s\" which has triggers ",
+							   relationName),
+						errdetail("Citus does not support distributing tables having "
```

Comment:
Afaik, currently yes, we do not support triggers

Reply:
A minor suggestion/edit for the error:

errmsg: cannot distribute relation \"%s\" because it has triggers, relationName
errdtl: Citus does not support distributing tables with triggers
errhint: Drop all the triggers on \"%s\"  and retry